tests/data/DatabaseConnection.ts

```diff
@@ -0,0 +1,49 @@
+import { Connection, createConnection } from 'typeorm';
+import { Config } from '../../config';
+import { models as entities } from '../../models';
+
+export class DatabaseConnection {
+  private static conn: Connection = null;
+
+  public static async connect(): Promise<Connection> {
+    if (!DatabaseConnection.conn) {
+      DatabaseConnection.conn = await createConnection({
+        type: 'postgres',
+        host: Config.database.host,
+        port: Config.database.port,
+        username: Config.database.user,
+        password: Config.database.pass,
+        database: Config.database.name,
+        entities,
+      });
+    }
+    return DatabaseConnection.conn;
+  }
+
+  public static async get(): Promise<Connection> {
+    return DatabaseConnection.connect();
+  }
+
+  public static async clear(): Promise<void> {
+    const conn = await DatabaseConnection.get();
+    await conn.transaction(async (txn) => {
+      const tableNames = [
+        'Activities',
+        'OrderItems',
+        'Orders',
+        'MerchandiseItemOptions',
+        'MerchandiseItems',
+        'MerchandiseCollections',
+        'Attendances',
+        'Users',
+        'Events',
+      ];
+      await Promise.all(tableNames.map((t) => txn.query(`DELETE FROM "${t}"`)));
+    });
```

Comment:
nit: `t` -> `tableName`/`table`

Reply:
`tableName` is a bit too close to the array `tableNames`, which's why I use the single letter for quick functions like this, otherwise with variables so similarly named it's easy to misread/misuse.